sdk/attestation/azure-security-attestation/README.md

```diff
@@ -404,7 +400,7 @@ ignored (this possibly surprising behavior is there because retries could cause
     {
       // Create a PEM encoded X.509 certificate to add based on the POLICY_SIGNING_CERTIFICATE_0
       // certificate.
-      std::string const certToAdd(GetEnvHelper::GetEnv("POLICY_SIGNING_CERTIFICATE_0"));
+      std::string const certToAdd(std::getenv("POLICY_SIGNING_CERTIFICATE_0"));
```

Comment:
Check for null value from return from std::getenv before assigning to a std::string.